### PR TITLE
Fix some typos for DataMovers type.

### PR DESCRIPTION
**Pull request by yanggangtony**
Thank you for contributing to Velero!

# Please add a summary of your change
There are some typos for the code notes ,fix it for the right semantics. 

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: config/crd/v2alpha1/bases/velero.io_datauploads.yaml [hunk 1]
```diff
@@ -79,7 +79,7 @@ spec:
                 nullable: true
                 properties:
                   snapshotClass:
-                    description: StorageClass is the name of the snapshot class that
```
Comment (Lyndon-Li) [inline config/crd/v2alpha1/bases/velero.io_datauploads.yaml:82]: Please run `make update` and commit all the changes, i.e., crds.go

File: config/crd/v2alpha1/bases/velero.io_datauploads.yaml [hunk 2]
```diff
@@ -79,7 +79,7 @@ spec:
...
+                    description: SnapshotClass is the name of the snapshot class that
```
Comment (Lyndon-Li) [inline config/crd/v2alpha1/bases/velero.io_datauploads.yaml:82]: Please run make update and commit all the changes, i.e., crds.go

Comment (Lyndon-Li) [inline config/crd/v2alpha1/bases/velero.io_datauploads.yaml:82]: Please run make update and commit all the changes, i.e., crds.go

**Review by Lyndon-Li — CHANGES_REQUESTED** [review]

File: config/crd/v2alpha1/bases/velero.io_datauploads.yaml (hunk 2 above)
Comment (yanggangtony) [reply config/crd/v2alpha1/bases/velero.io_datauploads.yaml:82]: Thanks .
I used to run it.
But it will update a lot files.
I will rebased later.

Comment (yanggangtony) [reply config/crd/v2alpha1/bases/velero.io_datauploads.yaml:82]: @Lyndon-Li 
Updated.

Comment (Lyndon-Li) [reply config/crd/v2alpha1/bases/velero.io_datauploads.yaml:82]: This still doesn't look correct:

- v1/crds/crds.go should not affected
- v2alpha1/crds/crds.go should not be that long

I suggest you rollback to the latest main code, do the changes again and then run `make update`

Comment (yanggangtony) [reply config/crd/v2alpha1/bases/velero.io_datauploads.yaml:82]: image

Comment (yanggangtony) [reply config/crd/v2alpha1/bases/velero.io_datauploads.yaml:82]: @Lyndon-Li 
Thanks for quick review.
I actually follow your suggestion , and run it.
The pic above shows that the result is both 
``` v1/crds/crds.go``` and  ```v2alpha1/crds/crds.go ``` had affected.

Is that a possible , the crd had been modified before, but not execute the 'make update'?

Comment (Lyndon-Li) [reply config/crd/v2alpha1/bases/velero.io_datauploads.yaml:82]: Yes, you are right. It should be due to this PR https://github.com/vmware-tanzu/velero/pull/6415.
Then I think we can fix this problem in this PR.

Thanks.

**Review by Lyndon-Li — APPROVED** [review]

**Comment by yanggangtony** [comment]
@reasonerjt 
friendly ping.
would you take a look at this pr?

**Review by reasonerjt — APPROVED** [review]

**Comment by reasonerjt** [comment]
Thanks @yanggangtony I'll merge this PR.

I don't check notification from Github quite often please ping me on Slack if you need more immediate response.

**Comment by yanggangtony** [comment]
@reasonerjt 
Thanks.
Actually this pr is not that urgent for me.
I will follow the issue and pr in the contents  , not in hurry.

**Pull request merged**